src/api/config/genesis-params.md

```diff
@@ -0,0 +1,50 @@
+# Genesis Parameters
+
+Explain the purpose of this block. Maybe explain both keys in a single
+section?
```

Comment:
this is a comment, not the content, so can you please have this (and similar instances) commented out?

Reply:
Great suggestion!

Reply:
Used TODOs